.gitlab-ci.yml

```diff
@@ -0,0 +1,8 @@
+update:
+  only:
+    - master
```

Comment:
I don't think we want this to run on commits to master since this job commits to master.

Reply:
good catch! removing now.